### PR TITLE
docs: update lambda setup docs acc. to console updates

### PR DESCRIPTION
**Pull request by Just-Moh-it**
### A few changes 🙃

All while AWS tries to fix its broken UX, we try to fix our outdated docs 😉

Screenshots relating to changes 👇

Screenshot 2023-02-05 at 9 29 49 PM
Screenshot 2023-02-05 at 9 32 52 PM
Screenshot 2023-02-05 at 9 32 58 PM
Screenshot 2023-02-05 at 9 36 46 PM
Screenshot 2023-02-05 at 9 39 26 PM
Screenshot 2023-02-05 at 9 46 27 PM

Screenshot 2023-02-05 at 9 29 08 PM

### CODE REVIEW

File: packages/docs/docs/lambda/setup.md [hunk 1]
```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
...
+
+- Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
+- Click on the name of the user that was created in step 4.
+- Navigate to the "Security Credentials" tab, and scroll down to the "Access Keys" section.
+- Click the "Create access key" button.
+- Select "Application running on an AWS compute service".
+- Ignore warnings that might appear and check the "I understand the recommendations..." checkbox.
+- Click "Next".
+- Click "Create access key".
+- Add a `.env` file to your project's root and add the credentials you just copied in the following format:
```
Comment (Just-Moh-it) [inline packages/docs/docs/lambda/setup.md:100]: AWS added a new step for finer control over access tokens 🥲 Hence the new steps

File: packages/docs/docs/lambda/setup.md [hunk 2]
```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
...
 
 ```txt title=".env"
 REMOTION_AWS_ACCESS_KEY_ID=<Access key ID>
 REMOTION_AWS_SECRET_ACCESS_KEY=<Secret access key>
 ```
 
-## 5. Add permissions to your user
+## 6. Add permissions to your user
```
Comment (Just-Moh-it) [inline packages/docs/docs/lambda/setup.md:107]: And... the step counter cascades to the end

File: packages/docs/docs/lambda/setup.md [hunk 3]
```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
...
 
 - Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
 - Select the user you just created.
-- Click "Add inline policy" on the right of the screen under "Permissions policies".
+- Click "Add inline policy" under the "Add Permissions" dropdown in the "Permissions policies" panel.
```
Comment (Just-Moh-it) [inline packages/docs/docs/lambda/setup.md:111]: > "right of the screen"

*Depends*. I was on a narrow version of AWS, so it defaulted to the mobile styles, and the option was no longer on the right. We should not add super-specific location specifications in the docs as it depends heavily on the layout and the viewport.

Also, the option was no longer on the right side of the screen 🙃

File: packages/docs/docs/lambda/setup.md [hunk 4]
```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
...
 - Click the tab "JSON".
 - Enter in your terminal: `npx remotion lambda policies user` and copy into the AWS text field what gets printed.
-- Give the policy a name. For example `remotion-user-policy`, but it can be anything..
+- Click "Review policy".
+- Give the policy a name. For example `remotion-user-policy`, but it can be anything.
```
Comment (Just-Moh-it) [inline packages/docs/docs/lambda/setup.md:115]: The previous docs missed a step about moving to the next screen. Added it 🎊

File: packages/docs/docs/lambda/setup.md [hunk 5]
```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
 - Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
 - Click `Add users`
 - Enter any username, such as `remotion-user`.
-- **Check** the "Access key - Programmatic access" option.
```
Comment (Just-Moh-it) [inline packages/docs/docs/lambda/setup.md:85]: This option is no longer available and has to be created as outlined in Step 5

**Comment by JonnyBurger** [comment]
Wow! Thanks for making these changes. I love it when something existing gets refined and you did it masterfully 🥲

**Review by JonnyBurger — APPROVED** [review]

**Pull request merged**